Xamarin.Forms.Platform.Android/CollectionView/SimpleViewHolder.cs

```diff
@@ -20,9 +20,26 @@ public void Recycle(ItemsView itemsView)
 			itemsView.RemoveLogicalChild(View);
 		}
 
+
+		public static SimpleViewHolder FromText(string text, Context context, Func<int> width, Func<int> height)
```

Comment:
This needs to account for orientation; the size adjustment for the Empty view might have to be width (for a horizontal layout). Try changing the 8326 test to a horizontal linear layout to see what I mean.

Reply:
Right, good point. We continue using the previous method (using `MatchParent`).